config/bundles.php

```diff
@@ -43,7 +43,7 @@
     Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle::class => ['all' => true],
     Sylius\Bundle\FixturesBundle\SyliusFixturesBundle::class => ['all' => true],
     Sylius\Bundle\PayumBundle\SyliusPayumBundle::class => ['all' => true],
-    Sylius\Bundle\ThemeBundle\SyliusThemeBundle::class => ['all' => true],
+//    Sylius\Bundle\ThemeBundle\SyliusThemeBundle::class => ['all' => true],
```

Comment:
If I understand it correctly we should leave the SyliusBundle active, but not on the `test` environment.

```suggestion
    Sylius\Bundle\ThemeBundle\SyliusThemeBundle::class => ['dev' => true, 'prod' => true, 'test' => false],
```

Reply:
default is false, yet i would like to highlight this decison